samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,134 @@
+.. _at_client_sample:
+
+nRF9160: AT Client
+##################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem.
+The sample enables you to use an external computer or MCU to send AT commands to the LTE-M/NB-IoT modem of your nRF9160 device.
+
+Overview
+********
+
+The AT Client sample acts as a proxy for sending directives to the nRF9160 modem via AT commands.
+This facilitates the reading of responses or analyzing of events related to the nRF9160 modem.
+The commands can be initiated from a terminal or the `LTE Link Monitor`_, which is an application implemented as part of `nRF Connect for Desktop`_.
+
+For more information on the AT commands, see the `AT Commands Reference Guide`_.
+
+
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/build_and_run_nrf9160.txt
+
+
+Connecting to the nRF9160 DK board with LTE Link Monitor
+========================================================
+
+To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
+
+#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
+#. Make sure that **Automatic requests** is enabled in LTE Link Monitor.
+#. Connect the nRF9160 DK board to the PC with USB.
+#. Power on the nRF9160 DK board.
+#. Click :command:`Select Device` and select the particular board device entry from the drop-down list in the LTE Link Monitor.
```

Comment:
In the output it looks the same, but this isn't a command and should be just bold.

Reply:
modified